### PR TITLE
Ruby: Unit 1, Lesson 4: Serialization & Deserialization

### PR DESCRIPTION
**Pull request by aviflax**

### CODE REVIEW

File: ruby/unit_1/lesson_4/fragment_1.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Unit 1, Lesson 3: Serialization and Deserialization
```
Comment (arsenerei) [inline ruby/unit_1/lesson_4/fragment_1.rb]: Lesson 4. 😀

File: ruby/unit_1/lesson_4/fragment_1.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+# Before we get started, please note: this lesson requires
+# that the network started by lesson 1 still be up and
+# running. So if you haven’t ran lesson 1 for awhile, you
+# should probably do so now, and then come back here.
+
+# We’ll use ruby-kafka again, and set it up the same way as
+# in lesson 2:
```
Comment (arsenerei) [inline ruby/unit_1/lesson_4/fragment_1.rb]: Lesson 2 or 3?

File: ruby/unit_1/lesson_4/fragment_1.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Unit 1, Lesson 3: Serialization and Deserialization
+
+# Before we get started, please note: this lesson requires
+# that the network started by lesson 1 still be up and
+# running. So if you haven’t ran lesson 1 for awhile, you
```
Comment (arsenerei) [inline ruby/unit_1/lesson_4/fragment_1.rb]: Is there a handy command we can provide in each lesson that checks if the lesson 1 network is up and if not, start it for them? I think that would keep them engaged in the current lesson.

File: ruby/unit_1/lesson_4/strings.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+Skate ipsum dolor sit amet, lien air slappy egg plant salad grind 540 Pantsman casper slide.
```
Comment (arsenerei) [inline ruby/unit_1/lesson_4/strings.txt]: 😆

File: ruby/unit_1/lesson_4/fragment_1.rb (hunk 1 above)
Comment (aviflax) [reply ruby/unit_1/lesson_4/fragment_1.rb]: 🤦‍♂️

File: ruby/unit_1/lesson_4/fragment_1.rb (hunk 2 above)
Comment (aviflax) [reply ruby/unit_1/lesson_4/fragment_1.rb]: both, actually. will fix.

File: ruby/unit_1/lesson_4/fragment_1.rb (hunk 3 above)
Comment (aviflax) [reply ruby/unit_1/lesson_4/fragment_1.rb]: Great idea! I’ve created #20 to track it.

**Pull request merged**